docs/quickstart.md

```diff
@@ -33,61 +37,57 @@ For more detailed installation requirements, please refer to [installation](/dep
 sudo docker logs walrus 2>&1 | grep "Bootstrap Admin Password"
 ```
 
-![qs-first-login](/img/v0.4.0/quickstart/qs-first-login-en.png)
+![qs-first-login](/img/v0.5.0/quickstart/qs-first-login-en.png)
 
-## Add a Kubernetes Cluster as Deployment Target
+3. After logging in, you can set a new password and configure the access address of Walrus.
 
-> Prerequisites:
->
-> - A Kubernetes cluster that can be reached by the Walrus server.
+![qs-set-pwd](/img/v0.5.0/quickstart/qs-set-pwd-en.png)
 
-1. Click the `Application Management` menu in the navigation bar to go to the `default` project view by default.
-2. Click the `Connectors` tab.
-3. Click the `New Connector` button, and select `Kubernetes` type.
-4. Fill in the cluster name and KubeConfig file, and click save.
+## Local Environment
 
-![qs-add-connector](/img/v0.4.0/quickstart/qs-add-connector-en.png)
+Walrus provides a default `local` environment under the `default` project for you to quickly experience the application deployment process. 
 
-## Create Environment
+![qs-local-env](/img/v0.5.0/quickstart/qs-local-env-en.png)
 
-1. Click the `Environment` tab, and click the `New Environment` button.
+The `local` environment connects to a local Kubernetes cluster connector created by built-in K3s. You can use it to deploy applications without having to configure a Kubernetes cluster.
+
+> Note:
+>
+> The embedded K3s cluster connector is only used for testing and is not recommended for production environments. If you need to deploy applications in a production environment, please refer to [HA setup guide](deploy/replication) or configure a Kubernetes cluster connector manually.
 
-![qs-create-env](/img/v0.4.0/quickstart/qs-create-env-en.png)
+![qs-local-connector](/img/v0.5.0/quickstart/qs-local-connector-en.png)
 
-2. Enter the environment name, such as `dev`.
-3. Click `Add Connector` button, and select the Kubernetes connector added in the previous step.
-4. Click save.
+## Deploy Resources
 
-![qs-create-env2](/img/v0.4.0/quickstart/qs-create-env2-en.png)
+1. In the environment list page, click the `local` environment to enter the environment view.
 
-## Deploy Service
+![qs-env-list](/img/v0.5.0/quickstart/qs-env-list-en.png)
 
-1. In the environment list page, click the environment name created in the previous step to enter the environment view.
+2. Click the `Import YAML` button to import the built-in Walrus File resource.
 
-![qs-go2env](/img/v0.4.0/quickstart/qs-go2env-en.png)
+![qs-create-res](/img/v0.5.0/quickstart/qs-create-res-en.png)
 
-2. Click the `New` button and select `Service`.
+3. Select the `nginx` resource from the Walrus File Hub resource list on the right and import the YAML.
```

Comment:
Select the `nginx` example from the Walrus File Hub on the right.
To avoid conflicting with Walrus resource

Reply:
updated